### ISSUE TITLE
fix documentation and directory for worked examples

### ISSUE THREAD

**Issue by kmcdono2** [Living-with-machines/MapReader#386]
Per JOSS review:
- [x] think about naming conventions for worked examples @kmcdono2 
- [x] move annotation and post processing notebooks to geospatial directory @rwood-97 
- [x] update readme in geospatial @kmcdono2 
   - [x] add list of worked examples to README for this directory and explain origins/task @kmcdono2 
- [x] add/update non-geospatial README @kmcdono2 
   - [x] add list of worked examples to README for this directory and explain origins/task @kmcdono2 
- [ ] add this list to documentation page for worked examples @rwood-97 
- [x] add explanation that you need to clone the repo to run the worked examples @rwood-97

**Comment by kmcdono2** [comment]
Old `README` from `worked_examples/geospatial`:

# MapReader and geospatial images

- [Tutorials](#tutorials)
  - [classification_one_inch_maps_001](./classification_one_inch_maps_001)
- [Why use MapReader?](#why-use-mapreader)
- [Guidance for specific user groups](#guidance-for-specific-user-groups)

## Tutorials

- [classification_one_inch_maps_001](./classification_one_inch_maps_001)
  * **Goal:** train/fine-tune PyTorch CV classifiers on historical maps.
  * **Dataset:** from National Library of Scotland: [OS one-inch, 2nd edition layer](https://mapseries-tilesets.s3.amazonaws.com/1inch_2nd_ed/index.html).
  * **Data access:** tileserver
  * **Annotations** are done on map patches (i.e., slices of each map).
  * **Classifier:** train/fine-tuned PyTorch CV models.

## Why use MapReader?

MapReader enables quick, flexible research with large map corpora. It is based on the patchwork method, e.g. scanned map sheets are preprocessed to divide up the content of the map into a grid of squares. Using image classificaion at the level of each patch allows users to define classes (labels) of features on maps related to their research questions.

### You might be interested in using MapReader if:
- you have access to a large corpus of georeferenced maps
- you want to quickly test different labels to help refine your research question before/without committing to manual vector data creation
- your maps were created before surveying accuracy reached modern standards, and therefore you do not want to create overly precise geolocated data based on the content of those maps

### MapReader is well-suited for finding spatial phenomena that:
- have a homogeneous visual signal across many maps
- may not correspond to typical categories of map features that are traditionally digitized as vector data in a GIS


## Guidance for specific user groups

### "I want to search the visual contents of a large set of maps to help me answer a question about the past."

- MapReader can help you find instances of spatial phenomena in a collection of maps that is too large for you to 'close read/view'.
- MapReader creates output that you can link and analyze in relation to other geospatial datasets (e.g. census, gazetteers, toponyms in text corpora).

### "I have a collection of maps that have been scanned and georeferenced. How can I use MapReader?"

- If your maps cannot be openly released, MapReader can be used to create derived data that can be shared publicly. The institution could create these datasets or individual researchers could create datasets specific to their research questions.

**Comment by kmcdono2** [comment]
NEW REVISED `README` from `worked_examples/geospatial`: 

# MapReader and geospatial images (maps or earth observation imagery)

## Worked Examples 

| Task Name  | Input Type | Link to Input Data | Brief Description of What Notebook Does | Output Type | Created By |
| ------------- | ------------- | ------------- | ------------- | ------------- | ------------- |
| Content Cell  | Content Cell  | Content Cell  | Content Cell  | Content Cell  | Content Cell  |
| Content Cell  | Content Cell  | Content Cell  | Content Cell  | Content Cell  | Content Cell  |


Would you like to add a new Worked Example? Open a ticket and pitch the contribution to the team!

**Comment by kmcdono2** [comment]
Naming conventions for worked examples:

Geospatial
- patch classification
- patch classification: context model
- text spotting

Non-Geospaial
- patch classification

**Comment by kmcdono2** [comment]
Ask @evangeline-corcoran to review updated non-geospatial worked examples README to add link to her paper, model, annotations, etc. when it's ready.

**Comment by kmcdono2** [comment]
@rwood-97 - just needs for you to do the last few bits in the bullet points at the top, and approve my pull requests! Let me know if there are decisions to take about prose in the docs worked examples page.
